src/lib.rs

```diff
@@ -27,10 +116,14 @@ pub fn metadata() -> &'static cargo_metadata::Metadata {
     &METADATA
 }
 
+/// Allows you to fetch informations of a package in the current crate
 pub fn package(name: &str) -> Option<&cargo_metadata::Package> {
     metadata().packages.iter().find(|x| x.name == name)
 }
 
+/// Lazily return the default build directory.
+///
+/// Default for `target/debug/dist` in debug mode and `target/release/dist` in release mode
```

Comment:
Laziness is a detail of implementation here. The user can see the return type is a static reference, they will understand.

```suggestion
/// Get the default build directory.
///
/// The default for debug build is `target/debug/dist` and `target/release/dist` for the release build.
```